stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -268,6 +268,26 @@ lazy_static! {
     ADD COLUMN height_in_tenure;
     "#.into(),
     ];
+
+    pub static ref NAKAMOTO_CHAINSTATE_SCHEMA_4: [&'static str; 2] = [
+    r#"
+        UPDATE db_config SET version = "7";
+    "#,
+    // Add a `signer_stats` table to keep track of how many blocks have been signed by each signer
+    r#"
+        -- Table for signer stats
+        CREATE TABLE signer_stats (
+            -- Signers public key
+            public_key TEXT NOT NULL,
+            -- Stacking rewards cycle ID
+            reward_cycle INTEGER NOT NULL,
+            -- Number of blocks signed during reward cycle
+            blocks_signed INTEGER DEFAULT 1 NOT NULL,
```

Comment:
Should we have this be `default 0`?

Reply:
I think you are right...I am gonna assume this was a copy paste error cause I can't think of a reason why you would default it to 1.

Reply:
Wait wait...I think 1 might be okay...it would only ever create an entry per signer public key which I think would imply you saw at least one signature, no?

Reply:
> I think would imply you saw at least one signature, no

Yeah, if there's an entry in the table for it, the count can't be zero